modules/swagger-codegen/src/main/resources/objc/model-body.mustache

```diff
@@ -13,7 +13,11 @@
   }
   return self;
 }
-
+{{#vars}}{{^vendorExtensions.x-unique-id-key}}{{/vendorExtensions.x-unique-id-key}}{{#vendorExtensions.x-unique-id-key}}
+-(NSString*)indexPropertyName {
+    return @"{{name}}";
+}
+{{/vendorExtensions.x-unique-id-key}}{{/vars}}
```

Comment:
Can you please elaborate a bit more on its usage? From the example above, it adds an additional property "indexPropertyName", right? Is that used by database/ORM to indicate whether it's key (in DB) or not?